core/src/main/resources/schema/app/20211103115000_CB-14656_create_stackpatch_table.sql

```diff
@@ -0,0 +1,24 @@
+-- // CB-14656
+-- Migration SQL that makes the change goes here.
+
+CREATE TABLE IF NOT EXISTS stackpatch
+(
+    id              bigint NOT NULL,
+    type            CHARACTER VARYING(255) NOT NULL,
```

Comment:
maybe an index to match `findByStackAndType` would be useful

Reply:
I agree, adding it